packages/components/src/tab-panel/index.tsx

```diff
@@ -93,7 +94,12 @@ export function TabPanel( {
 		[ onSelect ]
 	);
 
-	const onNavigate = ( _childIndex: number, child: HTMLButtonElement ) => {
+	// Simulate a click on the newly focused tab, which causes the component
+	// to show the `tab-panel` associated with the clicked tab.
+	const activeTabAutomatically = (
```

Comment:
Nit: naming seems off here, did we mean to say `activateTabAutomatically`?

Reply:
Yup! Done in https://github.com/WordPress/gutenberg/pull/46004/commits/8ef6ea4fe0c64949ad0a4a30d48a0560ae37d498